protostar/commands/test/starkware/test_execution_state.py

```diff
@@ -36,7 +38,15 @@ async def from_test_suite_definition(
             contract=contract,
             starknet_compiler=starknet_compiler,
             context=TestContext(),
+            output_recorder=OutputRecorder(),
         )
 
+    def get_output(self, name: OutputName) -> str:
+        return self.output_recorder.captures[name].getvalue()
```

Comment:
I think the fact that this method exists here is bad, please remove it and adjust the code to my other comments.